client/src/includes/tabs.js

```diff
@@ -363,13 +363,13 @@ class Tabs {
 
 export default Tabs;
 
-export const initTabs = (tabs = document.querySelectorAll('[data-tabs]')) => {
```

Comment:
Best to just remove this file completely. Especially if you feel the PR is ready for review.